pkg/api/source/grpc_adapter.go

```diff
@@ -23,13 +25,50 @@ type (
 	StreamInput struct {
 		// Configs is a list of Source configurations specified by users.
 		Configs []*Config
```

Comment:
Right now there is no option that we will get a list of configs as we start multi-processes. When we will change do documentation and simplify this logic here?

Should we also do that for `executors`? 
Maybe we should do that as a follow-up after RBAC? WDYT @pkosiec @huseyinbabal?

Reply:
I can create a followup task for this, on my list. I also want to unify that, didn't wanted to confuse by having single config in sources and multiple configs in executors. Noted followup task.

Reply:
Followup https://github.com/kubeshop/botkube/issues/1001